packages/pymedphys_labs/src/pymedphys_labs/pedromartinez/qc-jaws.py

```diff
@@ -0,0 +1,1464 @@
+
+
+    # else:
+    #     fig, peak_figs, junctions_figs = merge_view_filtrot(ArrayDicom, dx, dy)
+    #
+    #     with PdfPages(dirname+'jaws_FR_report.pdf') as pdf:
+    #         pdf.savefig(fig)
+    #         for i in range(0, len(peak_figs)):
+    #             pdf.savefig(peak_figs[i])
+    #
+    #         for i in range(0, len(junctions_figs)):
+    #             pdf.savefig(junctions_figs[i])
+    #
+    #         plt.close()
+
+        # # if the platform is linux save the file in this folder
+        # if sys.platform == 'linux':
+        #     doc = fitz.open(dirname+'jaws_X_report.pdf')  # open the PDF
+        #     rect = fitz.Rect(0, 0, 100, 32)  # where to put image: use upper left corner
+        #
+        #     for page in doc:
+        #         page.insertImage(rect, filename="/mnt/home/peter/Dropbox/PhDMedPhysi/scripts-medphys/ahs-logo.png")
+        #
+        #     doc.saveIncr()
+        #
+        # # if the platform is windows save the file in this folder
+        # elif sys.platform == 'windows':
+        #
+        #     doc = fitz.open(dirname+'jaws_X_report.pdf')  # open the PDF
+        #     rect = fitz.Rect(0, 0, 100, 32)  # where to put image: use upper left corner
+        #
+        #     for page in doc:
+        #         page.insertImage(rect, filename="E:\zDropbox\Dropbox\PhDMedPhysi\scripts-medphys\\ahs-logo.png")
+        #
+        #     doc.saveIncr()
+        #
+        # # plt.show(block=True)  #this shows all the figures!! (most important plt.show)
+
+
+
+
+
+
+    # Normal mode:
+    print()
+    print("Directory folder.........:", dirname)
+    print("Storage type.....:", dataset.SOPClassUID)
+    print()
+
+    pat_name = dataset.PatientName
+    display_name = pat_name.family_name + ", " + pat_name.given_name
+    print("Patient's name...:", display_name)
+    print("Patient id.......:", dataset.PatientID)
+    print("Modality.........:", dataset.Modality)
+    print("Study Date.......:", dataset.StudyDate)
+    print("Gantry angle......", dataset.GantryAngle)
+    #
+    # if 'PixelData' in dataset:
+    #     rows = int(dataset.Rows)
+    #     cols = int(dataset.Columns)
+    #     print("Image size.......: {rows:d} x {cols:d}, {size:d} bytes".format(
+    #         rows=rows, cols=cols, size=len(dataset.PixelData)))
+    #     if 'PixelSpacing' in dataset:
+    #         print("Pixel spacing....:", dataset.PixelSpacing)
+    #
+    # # use .get() if not sure the item exists, and want a default value if missing
+    # print("Slice location...:", dataset.get('SliceLocation', "(missing)"))
+
+    # print(type(dataset))
+    # print(dataset.pixel_array)
+    # # exit(0)
+    # # plot the image using matplotlib
+    # plt.imshow(dataset.pixel_array, cmap=plt.cm.bone)
+    # plt.xlabel('X Distance from reference point')
+    # plt.ylabel('Y Distance from reference point')
+    # plt.show(block=True)
+
+
+try:

```

Comment:
From here down needs to be wrapped in a function called `main()` and then at the very bottom of the code you'll need the following:

```python
if __name__ == "__main__":
    main()
```

See an example within https://github.com/pymedphys/pymedphys/blob/master/scripts/test-watch.py

This is so that when tools load up your Python file just to look inside they're not forced into running your script.